modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- The floating toolbar would not be fully visible when the editor was placed inside
+  a scrollable container. #TINY-10335
```

Comment:
I don't think markdown works like that, only YAML 😂

Reply:
Oops, grabbed that from changie yaml file because I thought it was just wrapping, but turns out I missed a line break.